docs/tutorials/analysis/1D/cta_sensitivity.ipynb

```diff
@@ -27,7 +27,14 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "execution": {
```

Comment:
Same problem here...any idea what we can do about this @Bultako ?

Reply:
I decided to remove the `execution` meta data keyword during stripping https://github.com/gammapy/gammapy/commit/370fc3e67b8ed2a573cf993f4310710cb737b5cc @registerrier Please rebase and update the PR. Thanks!